doc/radosgw/config-ref.rst

```diff
@@ -954,6 +954,25 @@ Keystone Settings
 :Default: None
 
 
+``rgw keystone admin tenant``
```

Comment:
It might be worth mentioning that this setting regards v2 of OpenStack Identity API (aka Keystone API). AFAIK in v3 ```tenant``` is referred as ```project```.